src/core/utils.js

```diff
@@ -69,6 +69,9 @@ export function createResourceHint(opts) {
       if ("as" in opts) {
         linkElem.setAttribute("as", opts.as);
       }
+      if (opts.corsMode) {
+        linkElem.crossOrigin = opts.corsMode;
+      }
```

Comment:
@deniak Any reason to not use the same as line 64 above?
```suggestion
      if (opts.corsMode || url.origin !== document.location.origin) {
        linkElem.crossOrigin = opts.corsMode || "anonymous";
      }
```

Reply:
I guess this the approach in this PR is more explicit. So, that's ok.